server/server.lua

```diff
@@ -0,0 +1,38 @@
+if Config.OldESX then
+	ESX = nil
+	TriggerEvent('esx:getSharedObject', function(obj) ESX = obj end)
+end
+
+RegisterNetEvent('esx:playerLoaded') -- When a player loads in, we can store some basic information about them locally
+AddEventHandler('esx:playerLoaded', function(playerId, xPlayer, isNew)
+	ESX.Players[playerId] = xPlayer.job.name
+    local zPlayer = ESX.GetPlayerFromId(playerId)

```

Comment:
Do you really need to define this here as `xPlayer` is passed via this event: `(playerId, xPlayer, isNew)`

You use xPlayer to trigger a notification below this.